components/menu/test/menu-checkbox.visual-diff.js

```diff
@@ -47,7 +47,7 @@ describe('d2l-menu checkbox', function() {
 	});
 
 	describe('selection behavior', () => {
-		this.afterEach(async() => {
+		afterEach(async() => {
```

Comment:
No idea why this was needed, but visual diff tests were failing with `this.afterEach is not a function`.

Reply:
Interesting - I guess since `afterEach` is global for Mocha, it was not needed.  Were they always failing, or did they start failing when the outermost `describe` callback was changed to an arrow function?

Reply:
Yeah you're right, that likely explains it.